shell/platform/darwin/ios/framework/Source/FlutterViewController.mm

```diff
@@ -1220,6 +1228,50 @@ - (void)invalidateTouchRateCorrectionVSyncClient {
   _touchRateCorrectionVSyncClient = nil;
 }
 
+#pragma mark - Stylus Events
+
+- (void)pencilInteractionDidTap:(UIPencilInteraction*)interaction API_AVAILABLE(ios(13.4)) {
+  auto packet = [self createAuxillaryStylusActionData];
```

Comment:
I think this failure
```
2023-02-10 13:00:50.880393-0800 IosUnitTests[97230:523827] *** NSForwarding: warning: object 0x7ff7b43dd258 of class 'nil' does not implement methodSignatureForSelector: -- did you forget to declare the superclass of 'nil'?
2023-02-10 13:00:50.881393-0800 IosUnitTests[97230:523827] *** NSForwarding: warning: selector (0x7fefbeb1ca00) for message '0^*\^H' does not match selector known to Objective C runtime (0x600000c044d0)-- abort
2023-02-10 13:00:50.882077-0800 IosUnitTests[97230:523827] *** NSForwarding: warning: object 0x7ff7b43dd258 of class 'nil' does not implement doesNotRecognizeSelector: -- abort
```
is because `-pencilInteractionDidTap:` is called on a partial mock, and OCMock is choking trying to figure out the smart pointer signature of `createAuxillaryStylusActionData`.

It passes if you go back to returning `flutter::PointerData` from this method (and update the tests)
https://github.com/flutter/engine/pull/39267#discussion_r1102090058

```objc
- (flutter::PointerData)createAuxillaryStylusActionData
```

Reply:
Interesting find. @LouiseHsu can you try not returning the smart pointer format? Just return the raw packet type should work.

Reply:
I tried returning the raw packet type in an this earlier commit https://github.com/flutter/engine/pull/39267/commits/1cdf0ec7ec0dfc74049c8195dffc9f830de3058b
but I ran into compile errors

Reply:
> I tried returning the raw packet type in an this earlier commit https://github.com/flutter/engine/commit/1cdf0ec7ec0dfc74049c8195dffc9f830de3058b

Did you paste the wrong commit?